### ISSUE TITLE
Setting popover issue

### ISSUE THREAD

**Issue by smolinari** [hawkeye64/quasar#2]
Hi,

Win10 Pro, Chrome latest version. 

When I open the settings, the popover first goes down, then goes up. It happens in normal and full screen modes. 

Scott

**Comment by hawkeye64** [comment]
Yes, the animation for QMenu needs to be a bit better. Hopefully Raz deals with this. Because the QMenu is a child of the settings cog and can grow (it always grows down, even when the anchor is on the bottom) the menu becomes partially hidden below the screen. Raz told me to call the updatePosition method of QMenu to have it adjust itself so it is all visible. Thus we have the current result of that.

This really isn't a QMediaPlayer issue, but one on the QMenu.
